setup.py

```diff
@@ -23,6 +23,7 @@
         'Framework :: Django :: 1.9',
         'Framework :: Django :: 1.10',
         'Framework :: Django :: 1.11',
+        'Framework :: Django :: 2.0b1',
```

Comment:
on second glance, this isn't a valid tag on PyPI and the release will get rejected because of that

https://pypi.python.org/pypi?%3Aaction=list_classifiers is the list, 2.0 is on there.

Reply:
@adamchainz, should I create another pull request where I change `2.0b1` to `2.0`?

Reply:
yeah if @jsocol hasn't

Reply:
Done -> https://github.com/jsocol/django-ratelimit/pull/131